docs/history.md

```diff
@@ -1,8 +1,18 @@
 # History
 
-## 3.70.0 (2022-11-29)
+## 3.71.0 (2022-12-20)
 
-Issues Fixed 🩴
+Critical Changes 🎉
+
+-   A few weeks ago we soft-launched a `load_data` feature that matched the dataset loaded to the `org_shape`. If you had discovered and used the `load_data` option called `org_shape_match_only`, you should switch to the new task because eventually we will remove the option and only use the new task. (#3455)
```

Comment:
What "new task"? `load_sample_data`?
```suggestion
-   A few weeks ago we soft-launched a `load_data` feature that matched the dataset loaded to the `org_shape`. If you had discovered and used the `load_data` option called `org_shape_match_only`, you should switch to the new task because eventually we will remove the option and only use the new `load_sample_data` task. (#3455)
```